bin/ci-test.sh

```diff
@@ -49,11 +49,15 @@ do
         for mongo_driver in "${MONGO_DRIVERS[@]}"; do
             echo "Running tests with MONGO_DRIVER=$mongo_driver"
             export MONGO_DRIVER=$mongo_driver
-            coveragefile=`echo "$pkg" | awk -F/ '{print $NF}'`
+            coveragefile=$(echo "$pkg" | awk -F/ '{print $NF}')
             show go test -timeout ${TEST_TIMEOUT} ${race} --coverprofile=${coveragefile}.cov -v ${pkg}
         done
     else
-        coveragefile=`echo "$pkg" | awk -F/ '{print $NF}'`
+        coveragefile=$(echo "$pkg" | awk -F/ '{print $NF}')
         show go test -timeout ${TEST_TIMEOUT} ${race} --coverprofile=${coveragefile}.cov -v ${pkg}
     fi
+
+    echo "Running tests with GOEXPERIMENT=boringcrypto"
+    export GOEXPERIMENT=boringcrypto
+    show go test -tags=boringcrypto -timeout ${TEST_TIMEOUT} ${race} --coverprofile=${coveragefile}.cov -v ${pkg}
```

Comment:
At the moment, all the crypto-related functions are within the `pumps` package, but this test is running over all the packages. Should we run it only for `pumps` package, or leave it as it is?